api/sshRenew.go

```diff
@@ -60,7 +60,7 @@ func (h *caHandler) SSHRenew(w http.ResponseWriter, r *http.Request) {
 
 	newCert, err := h.Authority.RenewSSH(ctx, oldCert)
 	if err != nil {
-		WriteError(w, errs.ForbiddenErr(err))
+		WriteError(w, errs.ForbiddenErr(err, "error signing ssh certificate"))
```

Comment:
`error renewing ssh certificate`? The reason I ask is because `error signing ssh certificate` could be misunderstood to be a specific error during signing. But really this is just a catchall for any error that occurred during `RenewSSH`.

Reply:
Fixed on 9fd147f